runtime/src/configs/mod.rs

```diff
@@ -570,8 +570,9 @@ impl Get<Perbill> for MinNotFullBlocksRatio {
 }
 
 parameter_types! {
-    pub const RandomChallengesPerBlock: u32 = 10;
-    pub const MaxCustomChallengesPerBlock: u32 = 10;
+    pub const RandomChallengesPerBlock: u32 = 10; // If changing this, also change `TotalMaxChallengesPerBlock`
+    pub const MaxCustomChallengesPerBlock: u32 = 10; // If changing this, also change `TotalMaxChallengesPerBlock`
+    pub const TotalMaxChallengesPerBlock: u32 = 10 + 10;
```

Comment:
We could maybe have `TotalMaxChallengesPerBlock` as a function of the other two so changes are updated automatically

Reply:
+1

Reply:
I tried, but it doesn't compile easily. But I worked around it now.